Simplenote/SPAppDelegate.m

```diff
@@ -242,7 +242,19 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
-- (void)applicationDidBecomeActive:(UIApplication *)application {
+- (void)applicationDidBecomeActive:(UIApplication *)application
+{
+    [self ensurePinlockIsDismissed];
```

Comment:
I really appreciate how you encapsulated the logic for the pinlock in a new method, rather than just adding the event logging call after that code.

Readbility++ 👌 👏